yaml2ics.py

```diff
@@ -68,32 +70,17 @@ def event_ics_from_yaml(event_yaml: dict) -> str:
         rrule_dtstart = rrule_dtstart + 'Z'
         rrule_rrule = [line for line in rrule_lines if line.startswith('RRULE')][0]
 
-        event_lines = str(event).split('\r\n')
+        event.extra.append(ics.ContentLine(rrule_rrule))
 
-        # Splice in the rrule
-        out = []
-        for line in event_lines:
-            out.append(line)
-
-            if line.startswith('DTSTART'):
-                out.append(rrule_rrule + 'Z')
-    else:
-        out = str(event).split('\r\n')
-
-    now_utc = datetime.utcnow()
-    utc_stamp = now_utc.isoformat(
-        timespec='seconds'
-    ).replace('-', '').replace(':', '') + 'Z'
-    out.insert(-1, f'DTSTAMP:{utc_stamp}')
-
-    return '\r\n'.join(out)
+    event.dtstamp = datetime.utcnow().replace(tzinfo=dateutil.tz.UTC)
+    return event
 
 
 def events_to_calendar_ics(events: dict) -> str:
     cal = ics.Calendar()
     for event in events:
-        cal.events.add(event)
-    return str(cal)
+        cal.events.append(event)
+    return cal.serialize()
```

Comment:
We may want to have this be `events_to_calendar` instead, and not do the serialize at the end.  It may give us some further freedom later on.

Reply:
I needed it for the next thing I was going to do anyway!  The string vs object mixup became more important, since you can't put string events into calendars anymore (it appears to work but then serialization error later).  I tried to make it more standard now, all objects until the end.